ElectronClient/gui/KeymapConfigScreen.tsx

```diff
@@ -0,0 +1,229 @@
+import * as React from 'react';
+const CommandService = require('lib/services/CommandService').default;
+const { DOMtoElectronAccelerator } = require('lib/KeymapUtils.js');
+const { themeStyle } = require('lib/theme');
+const { _ } = require('lib/locale.js');
+// const { bridge } = require('electron').remote.require('./bridge');
+// const Setting = require('lib/models/Setting');
+
+interface KeymapItem {
+	label: string,
+	accelerator: string,
+	command: string
+	isEditing: boolean,
+}
+
+// Temporary values
+const commandNames = [
+	'textCopy',
+	'textCut',
+	'textPaste',
+];
+const getAccelerator = (commandName: string) => {
+	switch (commandName) {
+	case 'textCopy':
+		return 'Ctrl+C';
+	case 'textCut':
+		return 'Ctrl+X';
+	case 'textPaste':
+		return 'Ctrl+V';
+	default:
+		throw new Error('Invalid command');
+	}
+};
+
+const initialKeymap = commandNames.map(commandName => {
+	return {
+		label: CommandService.instance().label(commandName),
+		accelerator: getAccelerator(commandName),
+		command: commandName,
+		isEditing: false,
+	};
+});
+
+const KeymapConfigScreen = (props: { theme: Object }) => {
+	const [keymap, setKeymap] = React.useState([...initialKeymap]);
+	const [filter, setFilter] = React.useState('');
+
+	const theme = themeStyle(props.theme);
+	const containerStyle = {
+		...theme.containerStyle,
+		padding: 10,
+		overflow: 'auto',
+	};
```

Comment:
Styles should be defined using `buildStyle()` (see other components for examples)